Description: Fix djangodocs Sphinx extension to work with latest Sphinx
Origin: upstream, https://code.djangoproject.com/changeset/16231
Bug-Debian: http://bugs.debian.org/643758
Forwarded: not-needed

--- a/docs/_ext/djangodocs.py	(revision 16223)
+++ b/docs/_ext/djangodocs.py	(revision 16231)
@@ -135,4 +135,5 @@
         self.body.append('(')
         self.first_param = 1
+        self.param_separator = node.child_text_separator
 
     def depart_desc_parameterlist(self, node):
